scripts/jenkins/cloud/ansible/roles/ardana_upgrade/tasks/_upgrade-clm.yml

```diff
@@ -20,17 +20,8 @@
   environment:
     ARDANA_INIT_AUTO: 1
 
-- name: Copy up input model upgrade script
-  copy:
-    src: "{{ cloud_release }}-input-model-upgrade"
-    dest: "~/cloud-input-model-upgrade"
-    mode: 0755
-  register: ansible_copy_input_model_upgrade
-
 - name: Run input model upgrade script
-  shell: >-
-    ~/cloud-input-model-upgrade
-  register: ansible_input_model_upgrade
+  script: "{{ role_path }}/files/{{ cloud_release }}-input-model-upgrade"
```

Comment:
one question, in the original "copy" section, no additional path information was required... but using the script module it is, I assume thats intentional/necessary? seems odd that "copy" and "script" would operate out of different base directories

Reply:
The copy command automatically looks under the {{role_path}}/files area, whereas the script command you need to specify the explicit path to use.